### PR TITLE
Remove old Home plugin

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/cli/templates/default-app/plugins/welcome/dev/index.tsx [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
```
Comment (marcuseide) [inline packages/cli/templates/default-app/plugins/welcome/dev/index.tsx:1]: I don't think we want these notice headers in the templates

File: packages/cli/templates/default-app/plugins/welcome/src/components/WelcomePage/WelcomePage.tsx [hunk 2]
```diff
@@ -73,12 +88,8 @@ const WelcomePage: FC<{}> = () => {
...
                 <Link href="https://github.com/spotify/backstage/blob/master/docs/getting-started/create-a-plugin.md">
                   creating a plugin
-                </Link>{' '}
-                or have a look in the code for the{' '}
-                <Link component={RouterLink} to="/home">
-                  Home Page
-                </Link>{' '}
-                in the directory "plugins/home-page/src".
+                </Link>
+                .
```
Comment (marcuseide) [inline packages/cli/templates/default-app/plugins/welcome/src/components/WelcomePage/WelcomePage.tsx:92]: Maybe rephrase the sentence above if there is only one thing to check out 😄

File: packages/cli/templates/default-app/plugins/welcome/dev/index.tsx (hunk 1 above)
Comment (stefanalund) [reply packages/cli/templates/default-app/plugins/welcome/dev/index.tsx]: Fixed!

**Comment by stefanalund** [comment]
@marcuseide fixed now, thanks!

**Review by freben — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]
